src/shared-components/dropdown/desktopDropdown.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```

Comment:
some of the stuff in here is still named "select". Should update to dropdown. Same comment with many of the other files